app/models/concerns/orchestration/compute.rb

```diff
@@ -28,7 +28,7 @@ def compute_object
 
   def compute_provides?(attr)
     return false if compute_resource.nil?
-    compute? && compute_resource.provided_attributes.key?(attr)
+    compute_resource.provided_attributes.key?(attr)
```

Comment:
we could get rid of the previous condition as well :
```suggestion
    compute_resource&.provided_attributes.key?(attr)
```

Reply:
There is already `return false if nil?` guard clause.

Reply:
I mean that we could drop that guard clause now that we don't check `compute &&`

Reply:
Oh, we would need to :
```suggestion
    !!compute_resource&.provided_attributes&.key?(attr)
```
I'd argue it's not nicer